Modules/Filtering/ImageFeature/test/itkLaplacianImageFilterTest.cxx

```diff
@@ -59,7 +59,7 @@ itkLaplacianImageFilterTest(int argc, char * argv[])
   ITK_TRY_EXPECT_NO_EXCEPTION(reader->Update());
 
   // Assign a spacing other than [1,1] for testing
-  itk::Vector<float, 2> spacing({ 0.5, 5.0 });
+  itk::Vector<float, 2> spacing({ { 0.5, 5.0 } });
```

Comment:
Perhaps get rid of the round parentheses?

Reply:
Thanks for the suggestion @Leengit. Seems to work for the CI builds: 9739170. Rebased on `master` as well.